src/components/StatusLabel.svelte

```diff
@@ -0,0 +1,42 @@
+<script>
+  export let level = 'info';
+</script>
+
+<style>
+
+.status-label {
+  font-size: var(--text-015);
+}
+
+.status-label span {
+  display: inline-block;
+  padding: 3px 9px 4px;
```

Comment:
```suggestion
  padding: var(--space-1h) var(--space-base) var(--space-1h);
```

Reply:
this might be controversial, but keeping these values closedly aligned to the spacing vars will help us in the long-term. Let me know if it looks weird like this. I would also be open to `calc(var(--space-base) + 1px)` if it feels necessary to have these pixels closer to what you have. I know it's kind of verbose but it feels right to stick to the css vars we're using, but maybe this is too strict. Open to counterargument here

Reply:
Yep agreed - visually the values that work best are not symmetric but they *should* have a variable base we can offset from.